Source/WebCore/html/canvas/GPUCanvasContextCocoa.mm

```diff
@@ -307,7 +307,8 @@ static float interpolateHeadroom(float headroomForLow, float headroomForHigh, fl
 RefPtr<ImageBuffer> GPUCanvasContextCocoa::surfaceBufferToImageBuffer(SurfaceBuffer)
 {
     // FIXME(https://bugs.webkit.org/show_bug.cgi?id=263957): WebGPU should support obtaining drawing buffer for Web Inspector.
-    if (!m_configuration)
+    auto renderBufferCount = m_configuration->renderBuffers.size();
```

Comment:
the old code implies that `m_configuration` might not exist so i think it should be
```cpp
if (!m_configuration || !m_configuration->renderBuffers.size())
```